cfme/tests/control/test_alerts.py

```diff
@@ -354,7 +354,7 @@ def test_alert_timeline_cpu(request, vm, set_performance_capture_threshold, prov
 
 
 @pytest.mark.provider(CANDU_PROVIDER_TYPES)
-def test_alert_snmp(request, appliance, provider, setup_snmp, setup_candu, vm, wait_candu,
+def test_alert_snmp(request, appliance, provider, setup_snmp, vm, setup_candu, wait_candu,
```

Comment:
**question** is there a sensible way to set up a provider in a way that can be shared by `vm` and `setup_candu`

having a magic dependency on order of fixtures means we are open certain kinds of human error that may be avoidable

Reply:
I think the right thing to do here is to add `vm` to the list of fixtures required by `setup_candu`.

Reply:
@jkrocil vm and setup_candu fixtures have different scopes. I agree it requires refactoring, but it should be done in a separate PR.